src/library/Uno.Material/Styles/Application/SelectionControlColors.xaml

```diff
@@ -13,35 +13,35 @@
 	</ResourceDictionary.MergedDictionaries>
 
 	<SolidColorBrush x:Key="SelectControlBackgroundColor"
-					 Color="{StaticResource PrimaryColor}" />
+					 Color="{ThemeResource PrimaryColor}" />
 	<SolidColorBrush x:Key="SelectControlIndicatorColor"
-					 Color="{StaticResource OnPrimaryColor}" />
+					 Color="{ThemeResource OnPrimaryColor}" />
 	<SolidColorBrush x:Key="SelectControlStrokeColor"
-					 Color="{StaticResource OnSurfaceColor}"
+					 Color="{ThemeResource OnSurfaceColor}"
 					 Opacity="0.74" />
 	<SolidColorBrush x:Key="SelectControlToggleOnButtonColor"
-					 Color="{StaticResource PrimaryColor}" />
+					 Color="{ThemeResource PrimaryColor}" />
 	<!-- TODO: This is the OnPrimaryDisabledBrush, remove when brush issue #41 is fixed -->
 	<SolidColorBrush x:Key="SelectControlToggleOnBackgroundColor"
 					 Color="#615B4CF5" />
 	<SolidColorBrush x:Key="SelectControlToggleOnFocusBackgroundColor"
-					 Color="{StaticResource PrimaryColor}"
+					 Color="{ThemeResource PrimaryColor}"
 					 Opacity="0.1" />
 	<!-- TODO: This is the OnSurfaceHoverBrush, remove when brush issue #41 is fixed without oPACITY -->
 	<SolidColorBrush x:Key="SelectControlToggleOffButtonColor"
-					 Color="#F5F5F5" />
+					 Color="{ThemeResource BackgroundColor}" />
 	<SolidColorBrush x:Key="SelectControlToggleOffHoverButtonColor"
-					 Color="{StaticResource OnSurfaceColor}"
+					 Color="{ThemeResource OnSurfaceColor}"
 					 Opacity="0.05" />
 	<SolidColorBrush x:Key="SelectControlToggleOffFocusButtonColor"
-					 Color="{StaticResource OnSurfaceColor}"
+					 Color="{ThemeResource OnSurfaceColor}"
 					 Opacity="0.08" />
 	<!-- TODO: This is the OnSurfaceDisabledBrush, remove when brush issue #41 is fixed -->
 	<SolidColorBrush x:Key="SelectControlToggleOffBackgroundColor"
 					 Color="#61000000" />
 	<!-- TODO: This is the OnPrimaryMediumBrush, remove when brush issue #41 is fixed -->
 	<SolidColorBrush x:Key="SelectControlToggleOnDisabledButtonColor"
-					 Color="#B6A8FB" />
+					 Color="{ThemeResource PrimaryVariantLightColor}" />
```

Comment:
this doesn't fit with the comment. Can you validate with @NicolasChampagne  ?

Reply:
true, possibly OnPrimaryMediumAlternativeLightBrush until the issue fix is merged